PTMCMCSampler/PTMCMCSampler.py

```diff
@@ -562,131 +576,71 @@ def PTMCMCOneStep(self, p0, lnlike0, lnprob0, iter):
 
             # hastings step
             diff = newlnprob - lnprob0 + qxy
-            if diff > np.log(np.random.rand()):
+            if diff > np.log(self.stream.random()):
 
                 # accept jump
                 p0, lnlike0, lnprob0 = y, newlnlike, newlnprob
 
                 # update acceptance counter
                 self.naccepted += 1
                 self.jumpDict[jump_name][1] += 1
-
         # temperature swap
-        swapReturn, p0, lnlike0, lnprob0 = self.PTswap(p0, lnlike0, lnprob0, iter)
+        if iter % self.Tskip == 0 and self.nchain > 1:
+            p0, lnlike0, lnprob0 = self.pt_swap(p0, lnlike0)
+        # swapReturn, p0, lnlike0, lnprob0 = self.PTswap(p0, lnlike0, lnprob0, iter)
 
         # check return value
-        if swapReturn != 0:
-            self.swapProposed += 1
-            if swapReturn == 2:
-                self.nswap_accepted += 1
+        # if swapReturn != 0:
+        #     self.swapProposed += 1
+        #     if swapReturn == 2:
+        #         self.nswap_accepted += 1
 
         self.updateChains(p0, lnlike0, lnprob0, iter)
 
         return p0, lnlike0, lnprob0
 
-    def PTswap(self, p0, lnlike0, lnprob0, iter):
```

Comment:
Why do you change the function signature here?  Particularly, why the name change.  I get that `lnprob0` and `iter` are not used, but the convention for this code is to always include all 4 options.  Maybe @jellis18 can comment on whether it's okay to break the convention.

Reply:
Reverted to the original signature (and docstring).